CONTRIBUTING.md

```diff
@@ -21,9 +21,13 @@ For big feature it's advised to raise an issue to discuss it first.
 
   *If using an M1 ("Apple silicon") processor, read [the special instructions below](#special-instructions-for-apple-silicon-cpus).*
 
-* On Windows: [.NET Framework](https://dotnet.microsoft.com/download/dotnet-framework) 4.6.2 or higher.
+* On Windows: 
+  - [.NET Framework](https://dotnet.microsoft.com/download/dotnet-framework) 4.6.2 or higher.
+  - [C++ CMake tools for Windows](https://learn.microsoft.com/en-us/cpp/build/cmake-projects-in-visual-studio?view=msvc-170#installation)
```

Comment:
note: I don't mind having this text, just saying it's not accurate - you just need `cmake` in path, it doesn't need to come from VS

Reply:
I'm happy to reword to make it more accurate. Mainly I just wanted to help new contributors understand:
1. They need CMake installed (and on the path)
2. How to install it if they don't have it

We could say, "you'll need it on the path... if you don't have it, one way to get it is by installing the C++ CMake tools for Windows". If there's an easier or lighter weight approach we could recommend, happy with that as well.